packages/dart/noports_core/lib/src/sshnpd/sshnpd_params.dart

```diff
@@ -315,6 +318,12 @@ class SshnpdParams {
           ' ip addresses.',
     );
 
+    parser.addFlag(
+      'clear-cached-pks',
+      help: 'Clear cached public keys',
+      hide: true,
```

Comment:
Hidden. Might be useful to leave this in. Not sure whether to make it non-hidden or not.